expo/expo
# [expo] Fix dev loading view insets

# Why

Fixes ENG-4718

# How

The native module for react-native-safe-area-context changed and so I updated how we reach into it to pull out the initial insets.

# Test Plan

- Load NCL on an iPhone with a notch / home indicator and make a change to see that the dev menu appears underneath the home indicator
- Apply this patch and notice it now is properly positioned to account for the home indicator

# Checklist
